Header header

float64 GpsTime

float64 latitude
float64 longitude
float64 altitude


float64 g_alt
float64 g_lon
float64 g_lat
float32 g_E
float32 g_N
float32 g_U
float32 g_yaw

float64 N_vel
float64 E_vel
float64 U_vel


float64 roll
float64 pitch
float64 yaw

int32 pos_type
int32 slo_status
int32 flag
int32 posType
int32 headType

float32 car_f_l
float32 car_f_r
float32 car_b_l
float32 car_b_r